docs/supported_ops.md

```diff
@@ -5169,6 +5169,138 @@ Accelerator support is described below.
 <td> </td>
 </tr>
 <tr>
+<td rowSpan="6">ElementAt</td>
```

Comment:
Seems too much change in this doc for just adding a new expression, could you take a double check ?

Reply:
This is correct we added in a feature to put header rows periodically in the table, and this is one of the side effects.